3ds Max/Max2Babylon/Exporter/BabylonExporter.Material.GLTF.cs

```diff
@@ -238,6 +238,7 @@ internal void ExportPhysicalBase(GltfMaterialDecorator maxDecorator, BabylonPBRM
                 babylonMaterial.metallic = maxDecorator.Metalness;
                 babylonMaterial.roughness = maxDecorator.Roughness;
                 babylonMaterial.emissive = maxDecorator.EmissionColor?.ToArray();
+                babylonMaterial.twoSidedLighting = maxDecorator.DoubleSided;
```

Comment:
I'm not sure Max's doubleSided means twoSidedLighting. glTF's double sided means `backfaceCulling = false` and `twoSidedLighting = true`. I'm not sure what Max's doubleSided means.